src/mkGraph.ts

```diff
@@ -0,0 +1,47 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { splitIfPanelExists } from './extension';
+
+export function mkGraph(context: vscode.ExtensionContext): (...args: any[]) => any {
```

Comment:
I think yesterday I didn't do it properly.
Can you please move one layer of function call back to `registerCommand` in `extension.ts` file?
So end result will look something like
```typescript
export function mkGraph(context: vscode.ExtensionContext) {
    const currentPanel = vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
```
and 
```typescript
vscode.commands.registerCommand('ide-prototype.mkGraph',
        () => mkGraph(context)
    ),
```
same for the mkMarkdown